src/common/types/integrations.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import { FormattedMessage } from 'react-intl';
+import { APP_INTEGRATION } from '../../constants';
+
+type AppIntegrationAPIMiniItem = {
+    id: string,
+    type: typeof APP_INTEGRATION,
```

Comment:
This `typeof` is the only real change in this PR (besides explicit imports). Same thing on line 52, plus a corresponding change int the constants file.